modules/lib/src/main/resources/assets/js/app/inputtype/ui/selector/image/ImageEditor.ts

```diff
@@ -2285,4 +2315,17 @@ export class ImageEditor
     private notifyImageError(event: UIEvent) {
         this.imageErrorListeners.forEach(listener => listener(event));
     }
+
+    private getEditResetLabel(): string {
+        const item = new ResponsiveItem(this);
```

Comment:
Use class property (`this.responsiveItem`) that you will create in `createStickyToolbar`